### PR TITLE
Upgrade: do not check container runtime version

### PR DESCRIPTION
**Pull request by ereslibre**
## Why is this PR needed?

Fixes: https://github.com/SUSE/avant-garde/issues/729

When upgrading do not check the full container runtime version.

cri-o releases are not up to speed with Kubernetes releases, and while
we have already a new Kubernetes version to upgrade to, cri-o may be
lagging behind yet.

Since the patchlevel version drift is tolerable, do not take
explicitly into account the exact version when determining if a node
is up to date when it comes to checking the container runtime version.

### CODE REVIEW

**Comment by ereslibre** [comment]
Had to force push, too long commit title :)

File: internal/pkg/skuba/kubernetes/versions.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ var (
 			},
 			ComponentsVersion: ComponentsVersion{
 				KubeletVersion:          "1.15.2",
-				ContainerRuntimeVersion: "1.15.2",
+				ContainerRuntimeVersion: "1.15.0",
```
Comment (lcavajani) [inline internal/pkg/skuba/kubernetes/versions.go]: would it be sufficient to just have this version set to `1.15.0` ?

Comment (ereslibre) [reply internal/pkg/skuba/kubernetes/versions.go]: @lcavajani not sure I'm following, it's set to `1.15.0`, am I missing something?

Comment (lcavajani) [reply internal/pkg/skuba/kubernetes/versions.go]: sorry I mean, would it have worked if we had here `1.15.0` from the beginning ?

Comment (ereslibre) [reply internal/pkg/skuba/kubernetes/versions.go]: Yes, it would, but we'd have risked this happening in the future...

**Comment by ereslibre** [comment]
> What is the likelihood that the same problem happens with etcd in the future?

Low, I'd say, as its versions are not strictly tied to Kubernetes' as cri-o ones.

File: internal/pkg/skuba/kubernetes/versions.go (hunk 1 above)
Comment (lcavajani) [reply internal/pkg/skuba/kubernetes/versions.go]: Thanks, I just wanted to be sure I understand.

**Pull request merged**